### ISSUE TITLE
-> formatting

### ISSUE THREAD

**Issue by NoahTheDuke** [kkinnear/zprint#262]
https://github.com/kkinnear/zprint/blob/f9031d6762e3c7dc557baadc1e9d3a842944cb8c/src/zprint/config.cljc#L318-L323

Is there a reason `->` is categorized as having a "body"? Given how it's used and how `->>` is formatted, seems like it should be `:noarg1` instead of `:noarg1-body`.

```clojure
(->
  aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa)

(->>
 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa)
```

**Comment by kkinnear** [comment]
Hmmm.  These are both "body" functions because the things in them are typically things that are executable statements of some sort instead of just "arguments".  Not that I'm a big fan of the whole "indent differently because of arguments and body forms" thing, having done a lot of work in Lisp long before I met Clojure.  All that notwithstanding, both of these forms are things that should be "body" forms as far as I can tell.   If you don't think so, please elaborate on why.  Thanks!

The difference between how `->` and `->>` are formatted appears to be a bug to me at this point.  I can easily reproduce this, and I'll work on getting to the bottom of it.  But I think that it is the `->>` that is wrong given the current default configuration, not the `->`.  

Thanks for pointing this out!

**Comment by kkinnear** [comment]
Yes, the formatting for `:force-nl-body` was incorrectly handled in the code.  It will be fixed in the next release (almost certainly `1.2.5`).  That is independent of whether or not these should (or should not be) "body" functions.

**Comment by NoahTheDuke** [comment]
Ah, excellent! Thank you.

**Comment by kkinnear** [comment]
This is fixed in `1.2.5`
